x-pack/legacy/plugins/apm/public/components/app/ApmIndicesPermission/index.tsx

```diff
@@ -16,30 +16,40 @@ import {
 import { i18n } from '@kbn/i18n';
 import React, { useState } from 'react';
 import styled from 'styled-components';
+import { IndexPrivileges } from '../../../../typings/es_schemas/raw/IndexPrivileges';
 import { FETCH_STATUS, useFetcher } from '../../../hooks/useFetcher';
 import { fontSize, pct, px, units } from '../../../style/variables';
 import { ElasticDocsLink } from '../../shared/Links/ElasticDocsLink';
 import { SetupInstructionsLink } from '../../shared/Links/SetupInstructionsLink';
 
-export const Permission: React.FC = ({ children }) => {
-  const [isPermissionPageEnabled, setIsPermissionsPageEnabled] = useState(true);
+export const ApmIndicesPermission: React.FC = ({ children }) => {
+  const [
+    isPermissionWarningDismissed,
+    setIsPermissionWarningDismissed
+  ] = useState(false);
 
-  const { data, status } = useFetcher(callApmApi => {
-    return callApmApi({
-      pathname: '/api/apm/security/permissions'
-    });
-  }, []);
+  const { data = {} as IndexPrivileges['index'], status } = useFetcher(
```

Comment:
You shouldn't have to manually type it. If it's setup correctly the type will be inferred.